module/VuFind/src/VuFind/SMS/Clickatell.php

```diff
@@ -79,14 +79,14 @@ public function text($provider, $to, $from, $message)
         try {
             $result = $this->client->setMethod('GET')->setUri($url)->send();
         } catch (\Exception $e) {
-            throw new MailException($e->getMessage());
+            throw new SMSException($e->getMessage(), SMSException::ERROR_UNKNOWN);
         }
         $response = $result->isSuccess() ? trim($result->getBody()) : '';
         if (empty($response)) {
-            throw new MailException('Problem sending text.');
+            throw new SMSException('Problem sending text.', SMSException::ERROR_RESPONSE_UNKNOWN);
```

Comment:
Now that the SMSException provides an appropriate user message, I think we can change this back to ERROR_UNKNOWN and eliminate the ERROR_RESPONSE_UNKNOWN type. There is no translation defined for 'Problem sending text.' so that should not be used for end-user display. (When I first brought this up, I assumed that it was translated and didn't realize that sms_error was the correct string to use for end users).